src/resolvers/TicketsResolver.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-console */
+import { Arg, Query, Resolver, Mutation } from 'type-graphql';
+import Ticket from '../entities/Tickets';
+import TicketsModel from '../models/Tickets';
+import TicketInput from '../inputs/TicketInput';
+import TicketInputUpdate from '../inputs/TicketInputUpdate';
+import IdInput from '../inputs/IdInput';
+// import UsersModel from '../models/Users';
+
+@Resolver()
+class TicketsResolver {
+  @Query(() => [Ticket])
+  async allTickets() {
+    try {
+      const getAllTickets = await TicketsModel.find();
+      return getAllTickets;
+    } catch (err) {
+      return console.log(err);
+    }
+  }
+
+  @Query(() => Ticket)
+  async getOneTicket(@Arg('id', () => String) ticketId: IdInput) {
+    try {
+      const getOneTicket = await TicketsModel.findById(ticketId);
+      return getOneTicket;
+    } catch (err) {
+      return console.log(err);
+    }
+  }
+
+  @Mutation(() => Ticket)
+  async addTicket(@Arg('ticketInput') ticketInput: TicketInput) {
+    try {
+      // const getAllUsers = await UsersModel.find();
```

Comment:
Same here

Reply:
Could have been useful if we have to change the add mutation, but OK